optuna/study/_optimize.py

```diff
@@ -134,6 +136,7 @@ def _optimize_sequential(
     time_start: Optional[datetime.datetime],
     progress_bar: Optional[pbar_module._ProgressBar],
 ) -> None:
+    _in_optimize_loop.set(True)
```

Comment:
Thank you for your review! I actually tried it once. However, `n_jobs` option was broken since `_in_optimize_loop` is a thread local object and `n_jobs` option spawns new threads.

Reply:
That makes sense. Thanks!